target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -38,6 +38,16 @@ comfast,cf-e110n-v2)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "76" "100"
 	;;
+	comfast,cf-e120a-v3)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth0"
+	ucidef_set_led_switch "wan" "WAN" "$boardname:green:wan" "switch0" "0x04"
+	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
```

Comment:
Already defined in dts so this wlan trigger can be dropped

Reply:
@981213 True, the LED will work as expected but it won't appear in /etc/config/system. Is it OK?